src/validator/FindPathValidator.cpp

```diff
@@ -206,7 +206,7 @@ Status FindPathValidator::allPairPaths() {
 
     auto* conjunct = ConjunctPath::make(
         qctx_, forward, backward, ConjunctPath::PathKind::kAllPaths, steps_.steps);
-    conjunct->setColNames({"_path"});
+    conjunct->setColNames({kPathStr});
```

Comment:
conjunct‘s column name is not the output column name, it is used internally, if changed,   datacollect executor should be changed Simultaneously